<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">ACS Cent Sci</span> <span class="iso-abbrev" title="iso-abbrev">ACS Cent Sci</span> <span class="publisher-id" title="publisher-id">oc</span> <span class="coden" title="coden">acscii</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">ACS Central Science</span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">2374-7943</span> <span class="issn-epub" tagx="issn" title="issn-epub">2374-7951</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">American Chemical Society</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6661861">6661861</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1021/acscentsci.9b00297">10.1021/acscentsci.9b00297</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
A Mixed Quantum Chemistry/Machine Learning Approach for the Fast and Accurate Prediction of Biochemical Redox Potentials and Its Large-Scale Application to 315 000 Redox Reactions </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jinich</span> <span class="given-names" tagx="given-names" title="given-names">Adrian</span> </span> <a href="#aff1">†</a> <a href="#aff2">‡</a> <a href="#notes1">#</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanchez-Lengeling</span> <span class="given-names" tagx="given-names" title="given-names">Benjamin</span> </span> <a href="#aff1">†</a> <a href="#notes1">#</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">Haniu</span> </span> <a href="#aff1">†</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harman</span> <span class="given-names" tagx="given-names" title="given-names">Rebecca</span> </span> <a href="#aff1">†</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">Alán</span> </span> <a href="#cor1">*</a> <a href="#aff3">§</a> <a href="#aff4">∥</a> <a href="#aff5">⊥</a> </span> <span class="citation_author_institution" id="aff1">[†], <span class="institution" title="institution">Harvard University </span> <span class="country" tagx="country" title="country">United States</span> </span> <span class="citation_author_institution" id="aff2">[‡], <span class="institution" title="institution">Weill−Cornell Medical College</span> <span class="country" tagx="country" title="country">United States</span> </span> <span class="citation_author_institution" id="aff3">[§], <span class="institution" title="institution">University of Toronto</span> <span class="country" tagx="country" title="country">Canada</span> </span> <span class="citation_author_institution" id="aff4">[∥], <span class="institution" title="institution">Vector Institute </span> <span class="country" tagx="country" title="country">Canada</span> </span> <span class="citation_author_institution" id="aff5">[⊥], <span class="institution" title="institution">Canadian Institute for Advanced Research (CIFAR) </span> <span class="country" tagx="country" title="country">Canada</span> </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span>Phone: <span class="phone" tagx="phone" title="phone">416-978-3564</span>. E-mail: <span class="email" tagx="email" title="email">alan@aspuru.com</span>. </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-06-06</span> </span> <span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2019-07-07</span> </span> <span class="volume" tagx="volume" title="volume">5</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">1199</span> <span class="lpage" tagx="lpage" title="lpage">1210</span> <span class="history" title="history"> <span class="received" title="received">received: 2019-03-24</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">American Chemical Society</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article published under an ACS AuthorChoice <a href="http://pubs.acs.org/page/policy/authorchoice_termsofuse.html">License</a>, which permits copying and redistribution of the article or any adaptations for non-commercial purposes. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p> <div class="graphic_UNKNOWN" /> </p> <p>A quantitative understanding of the thermodynamics of biochemical reactions is essential for accurately modeling metabolism. The group contribution method (GCM) is one of the most widely used approaches to estimate standard Gibbs energies and redox potentials of reactions for which no experimental measurements exist. Previous work has shown that quantum chemical predictions of biochemical thermodynamics are a promising approach to overcome the limitations of GCM. However, the quantum chemistry approach is significantly more expensive. Here, we use a combination of quantum chemistry and machine learning to obtain a fast and accurate method for predicting the thermodynamics of biochemical redox reactions. We focus on predicting the redox potentials of carbonyl functional group reductions to alcohols and amines, two of the most ubiquitous carbon redox transformations in biology. Our method relies on semiempirical quantum chemistry calculations calibrated with Gaussian process (GP) regression against available experimental data and results in higher predictive power than the GCM at low computational cost. Direct calibration of GCM and fingerprint-based predictions (without quantum chemistry) with GP regression also results in significant improvements in prediction accuracy, demonstrating the versatility of the approach. We design and implement a network expansion algorithm that iteratively reduces and oxidizes a set of natural seed metabolites and demonstrate the high-throughput applicability of our method by predicting the standard potentials of more than 315 000 redox reactions involving approximately 70 000 compounds. Additionally, we developed a novel fingerprint-based framework for detecting molecular environment motifs that are enriched or depleted across different regions of the redox potential landscape. We provide open access to all source code and data generated. </p> </div> <div class="abstract" title="abstract"> <p>A combination of quantum chemistry and machine learning accurately predicts biochemical redox potentials of more than 300 000 reactions, revealing structural factors contributing to energetics. </p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="document-id-old-9" value="oc9b00297" /> <meta name="document-id-new-14" value="oc-2019-002977" /> <meta name="ccc-price" value="" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>All living systems are sustained by complex networks of biochemical reactions that extract energy from organic compounds and generate the building blocks that make up cells. <sup> <a href="#ref1">1</a> </sup> Recent work <sup> <a href="#ref2">2</a>− <a href="#ref5">5</a> </sup> has revived decades-old efforts <sup> <a href="#ref6">6</a>− <a href="#ref8">8</a> </sup> to obtain a quantitative understanding of the thermodynamics of such metabolic networks. Accurately predicting the thermodynamic parameters, such as Gibbs reaction energies and redox potentials, of biochemical reactions informs both metabolic engineering applications <sup> <a href="#ref9">9</a>, <a href="#ref10">10</a> </sup> and the discovery of evolutionary design principles of natural pathways. <sup> <a href="#ref11">11</a> </sup> This applies both to the prediction of the thermodynamics of known metabolic reactions and to non-natural reactions, which can be used to expand nature’s metabolic toolkit. <sup> <a href="#ref12">12</a>, <a href="#ref13">13</a> </sup> </p> <p>Experimentally available reaction Gibbs energies and redox potentials provide coverage for only about ∼10% of known natural metabolic reactions. The metabolic modeling community relies mainly on group contribution method (GCM) approaches <sup> <a href="#ref14">14</a>− <a href="#ref18">18</a> </sup> to estimate missing thermodynamic values. GCM decomposes metabolites into functional groups and assigns group energies by calibration against experimental data. The most widely used implementation of the GCM for biochemistry is the eQuilibrator, <sup> <a href="#ref19">19</a>, <a href="#ref20">20</a> </sup> an online thermodynamics calculator. In addition to using GCM to predict the formation energies of compounds, the eQuilibrator makes use of experimental reactant formation energies when available and combines them with group energies in a consistent manner in what is known as the component contribution method. This results in significant increases in accuracy. <sup> <a href="#ref18">18</a> </sup> However, whenever experimental reactant formation energies are not available, estimates are based solely on group contribution energies. Such GCM-based estimates, which do not capture intramolecular functional group interactions, have limited prediction accuracy. <sup> <a href="#ref21">21</a>, <a href="#ref22">22</a> </sup> </p> <p>In this work, we focus on predicting the thermodynamics of redox biochemistry. Redox reactions, which are fundamental to living systems and are ubiquitous throughout metabolism, consist of electron transfers between two or more redox pairs or half-reactions. <sup> <a href="#ref23">23</a> </sup> Quantum chemistry has recently emerged as an important alternative modeling tool for the accurate prediction of biochemical thermodynamics. <sup> <a href="#ref21">21</a>, <a href="#ref22">22</a>, <a href="#ref24">24</a>, <a href="#ref25">25</a> </sup> However, quantum chemical methods tend to have very high computational cost in comparison with the GCM or other cheminformatic-based alternatives. </p> <p>Recent work in the intersection of quantum chemistry and machine learning has resulted in hybrid approaches that significantly lower computational cost without sacrificing prediction accuracy. <sup> <a href="#ref26">26</a>− <a href="#ref29">29</a> </sup> One such hybrid quantum chemistry/machine learning approach, previously applied to organic photovoltaics material design, <sup> <a href="#ref30">30</a>, <a href="#ref31">31</a> </sup> relies on Gaussian process (GP) regression <sup> <a href="#ref32">32</a> </sup> to calibrate quantum chemical predictions against experimental data. GP regression is an established probabilistic framework in machine learning to build flexible models, which also furnishes uncertainty bounds on predicted data points. Gaussian process regression uses the "kernel trick" <sup> <a href="#ref33">33</a> </sup> to make probabilistic predictions, leveraging the distance between a data point of interest and a training set. </p> <p>Here, we present a mixed quantum chemistry/machine learning approach for the accurate and high-throughput prediction of biochemical redox potentials. We focus on predicting the thermodynamics of carbonyl (C=O) functional group reductions to alcohols (C—O) or amines (C—N) ( <a href="#fig1">Figure </a><a href="#fig1">1</a> A), two of the most abundant redox reaction categories in metabolism, for which a significant amount of experimental data is available. The approach relies on predicting electronic energies of compounds using the semiempirical PM7 method ( <a href="#fig1">Figure </a><a href="#fig1">1</a> B) <sup> <a href="#ref34">34</a> </sup> and then correcting for systematic errors in the predictions with GP regression against experimental data ( <a href="#fig1">Figures </a><a href="#fig1">1</a> C and <a href="#fig2">2</a>). We innovate on existing GP regression techniques by making use of novel reaction fingerprints <sup> <a href="#ref35">35</a> </sup> to compute the similarity between redox reactions. We demonstrate that the method results in higher predictive power than the GCM approach. Importantly, this comes at significantly lower computational cost than previously developed quantum chemical methodologies for redox biochemical thermochemistry. <sup> <a href="#ref22">22</a> </sup> In addition, we demonstrate the wide applicability of the GP regression method by directly calibrating GCM and fingerprint-based predictions (without quantum chemistry), also obtaining significant improvements in prediction accuracy. We demonstrate the high-throughput nature of the approach by predicting the redox potentials of what is to our knowledge the largest existing database of non-natural biochemical redox reactions, consisting of more than 315 000 iterative oxidations and reductions of "seed" metabolites from the KEGG database ( <a href="#fig1">Figure </a><a href="#fig1">1</a> A). <sup> <a href="#ref36">36</a>− <a href="#ref39">39</a> </sup> Using this vast data resource, we apply a novel molecular fingerprint-based analysis—termed "reaction fingerprint heatmap"—to decipher the molecular environment motifs associated with high and low redox potentials, thus providing insights into important structure–energy relationships. We provide open access to the full source code and data, including metabolite geometries, electronic energies, p <i>K</i> <sub>a</sub>’s, and redox potentials at <span class="uri" title="uri"> <a href="https://github.com/aspuru-guzik-group/gp_redox_rxn">https://github.com/aspuru-guzik-group/gp_redox_rxn</a> </span>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Schematic representation of the workflow. (A) We generate a network of redox reactions obtained from iterative reduction and oxidation of natural seed metabolites. The redox reactions considered are reductions of carbonyl functional groups to both alcohol and amine groups. (B) Starting from simplified molecular-input line-entry system (SMILES) <sup> <a href="#ref40">40</a> </sup> string representations of all the compounds in the network at pH = 0, we preoptimize molecular geometries using the Universal Force Field, and run semiempirical quantum chemistry calculations using PM7. (C) We estimate the standard (millimolar state) redox potentials <i>E</i> <sup>m</sup> for the major species at pH = 0 from the difference in electronic energies. The transformed standard redox potentials <i>E</i>′ <sup>m</sup>(pH = 7) is obtained from cheminformatic p <i>K</i> <sub>a</sub> estimates and the Alberty–Legendre transform. We calibrated a training set of compounds against available experimental <i>E</i>′ <sup>m</sup>(pH = 7) values using Gaussian process (GP) regression. </p>   </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="optimizationofredoxpotentialpredictionstrategythatcombinessemiempiricalquantumchemistrywithgpregression" title="sec"> <div class="title" tagx="title" title="title">
Optimization of Redox Potential Prediction Strategy That Combines Semiempirical Quantum Chemistry with GP Regression </div> <p>Our goal is to develop a calibrated quantum chemistry modeling framework that can accurately predict the reduction potentials of biochemical redox reactions in a high-throughput manner. Several different types of redox reactions that change the oxidation state of carbon atoms exist in biochemistry. These include the reduction of carboxylic acids to aldehydes, the reduction of carbonyls to alcohols or amines, and the reduction of alcohols to hydrocarbons. <sup> <a href="#ref4">4</a> </sup> In this work, we focus our efforts on predicting the standard redox potentials (as a function of pH and ionic strength) for the reduction of carbonyls to alcohols or amines. This reaction category represents the most common type of carbon redox transformation in biochemistry. <sup> <a href="#ref39">39</a> </sup> In addition, it is the category for which the largest amount of experimental data is available in the NIST database for the thermodynamics of enzyme-catalyzed reactions (TECRDB). <sup> <a href="#ref40">40</a> </sup> </p> <p>Similar to previous implementations, our approach consists of running quantum chemistry calculations to obtain electronic structure energies of substrates and products for the redox reactions of interest. <sup> <a href="#ref22">22</a> </sup> We take the difference in electronic energies as an estimate of the standard redox potential of the most abundant species (protonation states) of metabolites at pH = 0. We use cheminformatic p <i>K</i> <sub>a</sub> estimates and the Alberty–Legendre transform <sup> <a href="#ref41">41</a> </sup> to convert the redox potentials to transformed standard redox potentials <i>E</i>′° (pH = 7) ( <a href="#sec4">Methods</a>). We then use GP regression to correct for systematic errors in the ab initio estimates ( <a href="#fig2">Figure </a><a href="#fig2">2</a> ) as well as p <i>K</i> <sub>a</sub> estimates that go into the model ( <a href="#sec4">Methods</a>). To test and optimize the accuracy of our modeling framework, we compared predicted potentials against a data set of 81 experimental redox potentials obtained from the NIST database of thermodynamics of enzyme-catalyzed reactions. <sup> <a href="#ref40">40</a> </sup> </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Schematic diagram of GP regression with reaction fingerprints. A redox reaction of interest <i>x</i>* is represented using a reaction fingerprint, which captures the molecular structure of both substrate (oxidized) and product (reduced). This reaction fingerprint is compared against the set of all redox reactions in the experimental data set using the covariance function of the Gaussian process <i>k</i>( <i>x</i>*, <i>x</i> <sub> <i>i</i> </sub>) (see the <a href="#sec4">Methods</a> section). The GP correction ( <i>E</i>′ <sup>m</sup> <sub>GP</sub>) to the quantum chemical prediction ( <i>E</i>′ <sup>m</sup> <sub>PM7</sub>) of the standard redox potential is then obtained from a similarity-based average of the correction terms for the experimental data set. </p>   </div> <p>We tested several different model chemistries to maximize the accuracy and minimize the computational cost of our methodology ( <a href="#sec4">Methods</a> section, <a href="#tbl1">Table </a><a href="#tbl1">1</a> , <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b00297/suppl_file/oc9b00297_si_001.pdf">Table S1</a>). In this context, a "model chemistry" consists of the combination of multiple modeling choices that go into running a quantum chemical simulation. These include the conformer generation strategy and geometry optimization procedure, the electronic structure method (e.g., density functional theory (DFT), wave function approaches, or semiempirical methods) as well as the water model used. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <div class="title" tagx="title" title="title">
Summary Statistics of Prediction Accuracy for Calibration with Gaussian Process Regression of Different Modeling Approaches <a href="#t1fn1">a</a> </div>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th> </th> <th> </th> <th> </th> <th>calibration/regression <hr /> </th> <th>linear fit</th> </tr> <tr> <th>approach</th> <th>geometry source</th> <th>average compute time (s) </th> <th>MAE (σ) (mV)</th> <th> <i>r</i> </th> <th> <i>R</i> <sup>2</sup> </th> <th> <i>R</i> <sup>2</sup> </th> </tr> </thead> <tbody> <tr> <td>GCM</td> <td>N/A</td> <td>0.01</td> <td>27.15 (24.04)</td> <td>0.58</td> <td>0.33</td> <td>0.33</td> </tr> <tr> <td>Rxn-FP + GP</td> <td>N/A</td> <td>0.01</td> <td>24.35 (23.44)</td> <td>0.65</td> <td>0.42</td> <td>N/A</td> </tr> <tr> <td>GCM + GP</td> <td>N/A</td> <td>0.01</td> <td>21.20 (20.50)</td> <td>0.75</td> <td>0.56</td> <td>0.33</td> </tr> <tr> <td>PM7 + GP</td> <td>opt.</td> <td>0.36</td> <td>22.47 (19.68)</td> <td>0.74</td> <td>0.54</td> <td>0.22</td> </tr> <tr> <td>HF-3c + GP</td> <td>opt.</td> <td>588</td> <td>21.07 (19.76)</td> <td>0.76</td> <td>0.57</td> <td>0.28</td> </tr> <tr> <td>DSD-PBEP86 + GP</td> <td>PM7 opt.</td> <td>1223</td> <td>21.15 (17.97)</td> <td>0.79</td> <td>0.61</td> <td>0.24</td> </tr> <tr> <td>DLPNO–CCSD(T)//cc-pVDZ + GP</td> <td>B3LYP conf.</td> <td>6111</td> <td>20.45 (19.97)</td> <td>0.76</td> <td>0.57</td> <td>0.47</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">a</span> <p>See the <a href="#sec4">Methods</a> section for detailed descriptions of each modeling approach (quantum model chemistry). Rxn-FP = reaction fingerprints; GCM = group contribution method; GP = calibration with Gaussian process regression; opt. = geometry optimization of a single geometric conformer with the same level of theory; conf. = geometry optimization of multiple geometric conformers at a given theory level. Regression statistics correspond to GP calibration using Morgan fingerprints of radius 6. Additionally we report the <i>R</i> <sup>2</sup> (coefficient of determination) obtained using calibration with linear regression to the experimental data. The statistics are MAE = mean absolute error; σ = standard deviation of the absolute error; <i>r</i> = Pearson <i>r</i>; <i>R</i> <sup>2</sup> = coefficient of determination; average compute time = estimated average computational time per electron, in seconds. Linear fit is without GP regression. </p> </div> </div> </div> <p>The GP regression-based calibration step depends on a particular choice of a covariance function, which in turn involves a distance or similarity measure between redox reactions or compounds. We found that the following combination of quantum model chemistry and kernel/distance function between reactions resulted in the most efficient (in terms of a trade-off between accuracy and computational cost) prediction strategy: the PM7 <sup> <a href="#ref34">34</a> </sup> semiempirical method for both geometry optimizations and single-point electronic energies, with the COSMO implicit solvation model, <sup> <a href="#ref42">42</a> </sup> and a reaction fingerprint obtained by taking the difference between counted Morgan fingerprint vectors of products and substrates, <sup> <a href="#ref35">35</a> </sup> with a kernel function that is a mixture of a squared exponential kernel, and a noise kernel ( <a href="#sec4">Methods</a>). We note that the model chemistry using the double-hybrid functional DSD-PBEP86 (based on a double-hybrid DFT functional) gives the best accuracy but at a higher computational cost. </p> </div> <div class="gp-calibratedquantumchemistryyieldshighpredictionaccuracyatalowcomputationalcost" title="sec"> <div class="title" tagx="title" title="title">
GP-Calibrated Quantum Chemistry Yields High Prediction Accuracy at a Low Computational Cost </div> <p>Importantly, our GP-calibrated semiempirical quantum chemistry method results in higher predictive power than the most commonly used approach, the GCM ( <a href="#tbl1">Table </a><a href="#tbl1">1</a> , <a href="#fig3">Figure </a><a href="#fig3">3</a> ). Using cross-validation to test prediction accuracy, the GP-calibrated PM7 approach results in a higher Pearson correlation coefficient ( <i>r</i>), a higher coefficient of determination ( <i>R</i> <sup>2</sup>), and a lower mean absolute error (MAE) than GCM predictions when tested against the set of experimental potentials ( <i>p</i> &amp;lt; 0.05, <a href="#sec4">Methods</a>). The GP calibration also returns a GP uncertainty for each predicted value, and we find that it linearly correlates with the absolute deviation from uncertainty ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b00297/suppl_file/oc9b00297_si_001.pdf">Figure S1</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Comparison of prediction accuracy for the group contribution method and quantum chemistry with GP regression. Scatter plots of experimental standard transformed redox potentials against predictions from several sources: (a) PM7, (b) linearly calibrated PM7, (c) group contribution method (GCM), and (d) GP-calibrated PM7. The experimental data consist of available standard transformed reduction potentials for carbonyl-to-alcohol and carbonyl-to-amine reductions. Data points in the scatter plot were obtained from a leave-one-out cross-validation (LOOCV) procedure, where the model is trained on all data points except the one point to be predicted. Note that we use the millimolar standard state, <i>E</i>′ <sup>m</sup>, where reactant concentrations are taken as 1 mM, since this is significantly closer to relevant physiological concentrations of metabolites in cells than 1 M. The legend in each plot contains summary statistics for the fit; MAE (mean absolute error) also has the associated standard deviation in parentheses. </p>   </div> <p>We note that the GCM predictions, as well as fingerprint-based predictions (without relying on any quantum chemistry calculations), can also be calibrated using GP regression, resulting in significant improvements in their prediction accuracy. Thus, regardless of the underlying modeling choice, the GP calibration approach is an accurate, computationally efficient, and widely applicable modeling framework for the high-throughput prediction of biochemical redox potentials. </p> <p>One of the advantages of our approach is its low computational cost ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b00297/suppl_file/oc9b00297_si_001.pdf">Table S1</a>) in comparison to other model chemistries. For instance, the PM7-GP-calibrated calculations are approximately 1000-fold faster than the DLPNO–CCSD(T) calibrated model chemistry, despite both approaches yielding comparable accuracies. We note that the pipeline involves estimating p <i>K</i> <sub>a</sub>’s and major protonation states at pH = 0 for all metabolites at a slight additional computational cost. However, this cost is also incurred by the GCM approach. </p> </div> <div class="generationofalargenetworkofnon-naturalredoxreactionsusinganovelnetworkexpansionalgorithm" title="sec"> <div class="title" tagx="title" title="title">
Generation of a Large Network of Non-Natural Redox Reactions Using a Novel Network Expansion Algorithm </div> <p>To demonstrate the high-throughput nature of our method, we implemented a redox reaction network expansion algorithm to generate what is, to our knowledge, the largest database of non-natural biochemical redox reactions ( <a href="#sec4">Methods</a>). The network expansion algorithm makes use of the RDKit cheminformatics software to iteratively apply simplified molecular-input line-entry system (SMILES) <sup> <a href="#ref35">35</a>, <a href="#ref43">43</a> </sup> reaction strings to natural metabolites in the KEGG database. We generate two types of redox reactions, implemented in both the reductive and oxidative directions. The first type is the reduction of carbonyls to alcohols (and the reverse oxidations); the second is the reduction of carbonyls to amines (and the reverse oxidations). Applying the cheminformatic transformation to metabolites in the KEGG database results in a first set of nearest-neighbor reduction or oxidation products. We then iteratively apply the reactions to the resulting set of products, until there are no more carbonyl functional groups to reduce (or alcohol or amine functional groups to oxidize). </p> <p>Although the KEGG database of natural metabolic compounds and reactions contains molecules with up to 135 carbon atoms (a peptidoglycan cell wall component is the molecule with the highest number of carbon atoms), <sup> <a href="#ref44">44</a> </sup> we limit our network expansion algorithm to molecules with 21 carbon atoms or less. This threshold is imposed in consideration of the size of the compounds in the training set; if experimental data were available for larger compounds, this methodology could easily be adapted. </p> <p> <a href="#tbl1">Table </a><a href="#tbl1">1</a>  shows the number and type of reactions that make up the network. It consists of ∼75 000 compounds connected by ∼315 000 reactions (reductions and oxidations). Of these reactions, 83% convert carbonyls to alcohols (or alcohols to carbonyls), while 17% convert carbonyls to amines (or amines to carbonyls). The large majority of the reactions (more than 80%) are oxidations of alcohols to carbonyls. Thus, the alcohol functional group is significantly more ubiquitous in the set of natural metabolites than either carbonyls or amine functional groups. The large majority of the reactions in the network—97%—stem from the recursive <i>oxidations</i> of seed natural metabolites, while only ∼3% come from recursive reductions. This is consistent with the notion that carbonyl functional groups can cause damage to macromolecules <sup> <a href="#ref45">45</a>, <a href="#ref46">46</a> </sup> and are thus kept at check in the cell. </p> <p>We analyzed the structure of the resulting network by looking at the distribution of compound sizes. <a href="#fig4">Figure </a><a href="#fig4">4</a> A shows the number of compounds in the networks as a function of their number of carbon atoms. The distribution shows three peaks corresponding to molecules with 6 <i>n</i> carbon atoms ( <i>n</i> = 1, 2, 3). These peaks reflect a large number of products that result from combinatorially oxidizing all alcohol groups in mono-, di-, and trisaccharides ( <a href="#fig4">Figure </a><a href="#fig4">4</a> B). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>The network of redox reactions obtained from iterative reductions and oxidations of natural metabolites. (A) Carbon number distribution for the full set of ∼70 000 compounds in the network. The peaks at multiples of 6 carbon atoms reflect the combinatorial oxidation of mono-, di-, and trisaccharides and their multiple alcohol functional groups. (B) Schematic diagram of a natural trisaccharide being iteratively oxidized at all possible alcohol functional groups. Blue circles depict sites of initial alcohol-to-carbonyl oxidation, and green circles depict a second oxidation site. </p>   </div> </div> <div class="high-throughputpredictionofredoxpotentialsandelucidationofstructure–energyrelationships" title="sec"> <div class="title" tagx="title" title="title">
High-Throughput Prediction of Redox Potentials and Elucidation of Structure–Energy Relationships </div> <p>Using our GP-calibrated semiempirical quantum chemistry method, we predicted the standard transformed redox potentials for the set of ∼315 000 reactions generated with the iterative network expansion algorithm. The resulting distribution of standard transformed redox potentials is shown in <a href="#fig5">Figure </a><a href="#fig5">5</a> . We note the use of the millimolar standard state, <i>E</i>′ <sup>m</sup>, instead of the more commonly used <i>E</i>′° (where reactant concentrations are taken as 1 M). This is a standard state that is commonly used in the field of biochemical thermodynamics since 1 mM is closer to the relevant physiological concentrations of metabolites in cells. <sup> <a href="#ref4">4</a> </sup> The resulting distributions for the two reaction categories peak at very close but slightly different values, with ⟨ <i>E</i>′ <sup>m</sup>⟩ ∼ −220 mV for carbonyl-to-alcohol reductions and ⟨ <i>E</i>′ <sup>m</sup>⟩ ∼ −235 mV for carbonyl-to-amine reductions. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Distribution of predicted standard redox potentials (pH = 7) for a set of ∼315 000 non-natural redox reactions. Predicted potentials were obtained by running semiempirical quantum chemical calculations ( <a href="#sec4">Methods</a>) and correcting for systematic errors using Gaussian process regression. The green distribution shows the <i>E</i>′ <sup>m</sup>(pH = 7) values for 53 095 reductions of carbonyls to amines. The blue distribution shows the <i>E</i>′ <sup>m</sup>(pH = 7) values for 261 903 reductions of carbonyls to alcohols. </p>   </div> <p>Importantly, our vast network of non-natural redox reactions and their associated redox potentials are amenable to structure–energy analyses to elucidate molecular structures that are enriched or depleted in reactions with high or low potentials. Toward this end, we developed a novel molecular fingerprint-based structure–energy analysis, which we term "reaction fingerprint heatmaps", to detect such reaction motifs ( <a href="#fig6">Figure </a><a href="#fig6">6</a> ). The main idea behind reaction fingerprint heatmaps is that, by comparing the <i>average</i> reaction fingerprints of reactions with different redox potential values, one can visually detect the structural patterns that correlate with energetics. More specifically, we first divide the distribution of predicted standard redox potentials into arbitrary discrete bins, in our case 100 bins. We then find, for each standard redox potential bin, the subset of reactions with potentials falling within that range and average their corresponding fingerprint vectors. We then extract structure–energy trends by looking at the changes in average fingerprints values across consecutive bins. When these changes are large for a particular bit, it signifies that a meaningful molecular environment (motif) is enriched or depleted at that particular bin (range) of redox potential values. For any particular reaction, we can reverse engineer the reaction fingerprint to obtain substructures that activate the particular bit. For example ( <a href="#fig6">Figure </a><a href="#fig6">6</a> ), we find that Bit-4 is enriched in reactions with standard potentials of −160 mV &amp;lt; <i>E</i>′ <sup>m</sup> &amp;lt; −100 mV. By mapping Bit-4 back to specific reactions, we find a molecular environment that corresponds to a dicarbonyl within a heterocyclic ring (see example reaction in <a href="#fig6">Figure </a><a href="#fig6">6</a> ) that is enriched in substrate molecules within that range of standard potentials. While, across all reactions, the difference between the substrate and product molecules is the reduction of a single carbonyl into a hydroxy carbon group, our reaction fingerprint heatmaps detect the molecular environments dictating the energetics associated with that transformation. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>Elucidating structure–energy relationships using a reaction fingerprint heatmap. The top right panel shows the distribution of predicted standard redox potentials (in the millimolar standard state) using our GP-calibrated quantum chemical approach. The bottom right panel shows a reaction fingerprint heatmap, where the <i>y</i>-axis corresponds to the bits in the reaction fingerprint vector averaged across all reactions falling within a redox potential bin (range of values). The heatmap highlights bits that have significantly different values (on average) as a function of redox potential. The example reactions on the left correspond to each of the significant reaction fingerprint bits. For example, Bit-4 is a structure enriched in substrate molecules of reactions with high values of standard redox potentials. </p>   </div> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>In this work, we developed a mixed quantum chemistry/machine learning modeling approach for the prediction of biochemical redox potentials of carbonyl functional groups. The method is based on calculating the electronic energy difference between substrates and products in redox reactions using the semiempirical quantum chemistry method PM7. <sup> <a href="#ref34">34</a> </sup> The raw quantum chemistry estimates are then calibrated against experiment using GP regression. </p> <p>We demonstrated that the method has better prediction power than the commonly used GCM approach. Furthermore, the computational resources required are significantly lower than previous quantum chemistry strategies for biochemical thermodynamics. The GCM decomposes compounds into discrete functional groups and assigns energies to each group by calibrating via linear regression against experimentally available Gibbs energies. It then estimates reaction energies as the group energy difference between products and substrates. As <a href="#fig3">Figure </a><a href="#fig3">3</a> A shows, GCM collapses a large fraction of the redox potentials of carbonyl compounds into a few sets of values. We also note that, in a recent variant of the GCM, group energies can be combined with experimental reactant Gibbs formation energies to significantly increase the accuracy of predictions, resulting in what is known as the CCM. <sup> <a href="#ref18">18</a> </sup> Since we seek to apply our modeling approach to predicting redox potentials of non-natural reactions, the CCM is not applicable to the vast majority of the reactions in our data set. However, we also found that the GP regression technique can be used to further calibrate both GCM and fingerprint-based predictions (without relying on quantum chemical calculations), resulting in significant improvements to the prediction accuracies of these two approaches. Thus, the decision on what modeling choice to use can be based on a balancing act between accuracy, computational cost, and the preferred underlying modeling baseline. For example, if one desires an accurate, computationally cheap approach that relies on an underlying electronic structure method, PM7+GP is an optimal choice. Alternatively, for more accurate results that rely on more expensive electronic structure theory, DSDPEP89 (based on a double-hybrid DFT functional) gives the best accuracy but at higher computational cost. Likewise, if a good baseline for your chemical application is GCM, at a very low cost and almost comparable accuracy GCM+GP will be a good modeling choice. GP calibration proves to be a versatile approach to improve the accuracy of a wide variety of prediction methods. </p> <p>We considered model chemistries that rely on a single conformation per molecule and also ensembles of conformers. In our case we did not observe a huge difference in predicted accuracy between both approaches. Other chemical applications and data sets should be tested to see if performance is impacted by conformer diversity. A single missed hydrogen bond in one reacting partner, but not the other, could easily introduce large errors in the predicted redox potential. </p> <p>Current work on GP calibration in computational chemistry relies on <i>molecular</i> fingerprints to compute the similarity between molecules of interest and compounds with experimental data. Our implementation is to our knowledge the first application of GP to reaction energy predictions using reaction fingerprints. We chose to use GP—as opposed to other machine learning techniques such as neural networks—on the basis of the amount of experimental data available. GPs are suited for this task since they grow in complexity according to the data; a GP is robust to overfitting since it penalizes complex models and is able to provide estimates of uncertainty on predictions. <sup> <a href="#ref32">32</a>, <a href="#ref47">47</a>, <a href="#ref48">48</a> </sup> In contrast, techniques such as deep neural networks—which have gained traction in recent machine learning applications to quantum chemistry <sup> <a href="#ref49">49</a>− <a href="#ref51">51</a> </sup>—are only applicable when much larger data sets are available. </p> <p>We focused our approach to the prediction of the standard redox potentials of carbonyl functional groups (i.e., reduction to alcohols or amines). The application of GP regression to other redox transformations—including the reduction of carboxylic acids to aldehydes and the reduction of alcohols to hydrocarbons—is limited by the availability of experimental data. This highlights the importance of a concerted effort to generate more experimental data for biochemical redox potentials in controlled conditions (such as pH, ionic strength, temperature, and buffer) to increase the predictive power and scope of calibrated quantum chemical approaches. </p> <p>We demonstrated the high-throughput nature of the methodology by generating a network of more than 315 000 non-natural biochemical reactions involving ∼70 000 compounds. Our algorithm is different from other network expansion algorithms previously applied to metabolic networks. By solely focusing on redox biochemistry, iterative application of the reactions converges to molecules that cannot be further reduced or oxidized. This contrasts with other network expansion algorithms <sup> <a href="#ref52">52</a>, <a href="#ref53">53</a> </sup> used to investigate scenarios related to the origins of life, where several types of chemical transformations are considered, including carbon-bond formation and cleavage. Other network expansion algorithms also start from natural seed compounds but only consider natural metabolites. <sup> <a href="#ref12">12</a> </sup> In contrast, our algorithm uses natural metabolites as seeds but results in the expansion to a vast network of non-natural compounds. </p> <p>Despite the improvements obtained over GCM with our method, the variance in the prediction accuracies across individual reactions is still significant. In practice, this spread places a limit on the confidence with which one can predict the redox potential of an individual reaction of interest. This is true, and to a larger degree, for other methods currently widely used, such as GCM. However, the main value of developing a method with lower average prediction error lies in our ability to analyze large sets of compounds, where average trends in the energies of different molecular substructures can be captured. We thus developed a novel structure–energy relationship analysis framework which we term reaction fingerprint heatmaps. These allow us to detect molecular environments that are enriched or depleted in reactions that fall within specific regions (bins) of the redox potential distribution. This could potentially be useful for future metabolic engineering applications: being able to pinpoint the exact molecular environments that correlate with redox potentials would allow the fine-tuning of the energetics of synthetic metabolites. We note that this approach captures large-scale structure–energy trends. An alternative approach, which is able to associate weights to individual atoms or subfragments, are the similarity maps as introduced by Riniker et al. <sup> <a href="#ref54">54</a> </sup> </p> <p>We envision several applications of our biochemical redox potential prediction methodology. One such application is studying the thermodynamic landscape of specific families of natural metabolic compounds that undergo combinatorial reductions (or oxidations) of carbonyl (or alcohol or amine) functional groups. One such family of compounds are the brassinosteroids and the oxylipins, <sup> <a href="#ref55">55</a>− <a href="#ref57">57</a> </sup> structurally diverse plant metabolites that play important roles in many physiological processes. In addition, in the context of drug metabolism, our methodology could be applied to obtain quantitative insights into the thermodynamics of redox transformations such as those mediated by the P450 (CYP) superfamily of enzymes. <sup> <a href="#ref58">58</a> </sup> </p> <p>To our knowledge, our database contains the largest set of geometric structures, electronic energies, p <i>K</i> <sub>a</sub> estimates, and redox potentials of natural metabolites and compounds related to these through oxidoreductive transformations. We make all the code and data sets generated in this work—including metabolite 3D geometries, electronic energies, p <i>K</i> <sub>a</sub>’s, and redox potential estimates—available as an open source repository at <span class="uri" title="uri"> <a href="https://github.com/aspuru-guzik-group/gp_redox_rxn">https://github.com/aspuru-guzik-group/gp_redox_rxn</a> </span>. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="generationofmostabundant(major)protonationstatesatph=0" title="sec"> <div class="title" tagx="title" title="title">
Generation of Most Abundant (Major) Protonation States at pH = 0 </div> <p>Our pipeline is based on performing electronic structure simulations of the most protonated species of each biochemical compound involved in every redox reaction of interest. We run the quantum chemistry simulations on the major protonation states at pH = 0 to avoid errors associated with the prediction of anionic compound energies. <sup> <a href="#ref21">21</a>, <a href="#ref59">59</a>, <a href="#ref60">60</a> </sup> We use the ChemAxon calculator plugin (Marvin 17.7.0, 2017, ChemAxon) <i>cxcalc majormicrospecies</i> to generate, for each substrate and product involved in a biochemical redox reaction of interest, the major microspecies at pH = 0. </p> </div> <div class="geometryoptimization" title="sec"> <div class="title" tagx="title" title="title">
Geometry Optimization </div> <p>Using the SMILES string representation <sup> <a href="#ref43">43</a> </sup> for each compound in the reaction network, we obtain the string corresponding to the most abundant (major) microspecies at pH = 0 using ChemAxon. This smiles string is then converted to an RDKit "mol" object. From here we obtain 3D structures based on two decisions. (1) Whenever we want a single 3D geometry, we generate an initial conformer with RDKit (using the EmbedMolecule command, which uses distance geometry to obtain initial coordinates for a molecule), and we optimize this conformer’s geometry Universal Force Field (UFF) as implemented in RDKit. <sup> <a href="#ref61">61</a> </sup> (2) If we want several conformers (to Boltzmann average their energies), we generate around 1000 random "smart" conformers via the ETKDG force field, <sup> <a href="#ref62">62</a> </sup> calculate their energies, and pick between 5 and 20 conformers (based on size) and on clustering of energies (using the Butina clustering algorithm). Depending on the baseline model chemistry, either we further refine the geometry of the PM7 <sup> <a href="#ref34">34</a> </sup> semiempirical method with the COSMO implicit solvation model <sup> <a href="#ref42">42</a> </sup> and then a quantum chemistry approach (PBE-3H or B3LYP), or we simply use the base PM7 or UFF geometry and perform single-point energy (SPE) calculations. </p> </div> <div class="single-pointenergyestimates" title="sec"> <div class="title" tagx="title" title="title">
Single-Point Energy Estimates</div> <p>We then compute the electronic energy <i>E</i> <sub>electronic</sub> of every compound involved in a given redox reaction using the PM7 semiempirical method <sup> <a href="#ref34">34</a> </sup> with the COSMO implicit solvation model <sup> <a href="#ref42">42</a> </sup> (see below for other single-point energy SPE model chemistries tested). For both reductions of carbonyl groups to alcohols and amines, we add a hydrogen molecule to the substrate side of the redox reaction (in the direction of reduction) and compute its electronic energy. In addition, to balance reductions of carbonyls to amine functional groups, we add an ammonia molecule to the substrate side of the reaction and a water molecule to the product side of the reaction (in the direction of reduction) and compute their electronic energies. </p> <p>We use the difference in the electronic energies of products and substrates, Δ <i>E</i> <sub>electronic</sub>, as an estimate of the standard redox potential <i>E</i>°(major species, MS, at pH = 0): </p><div class="disp-formula" title="disp-formula"> <div class="graphic_UNKNOWN" /> </div>where <i>n</i> is the number of electrons (2 for all reactions considered here), and <i>F</i> is Faraday’s constant. <p /> <p>Our approach ignores the contribution of ro–vibrational enthalpies and entropies to Gibbs reaction energies. This significantly reduces the computational cost associated with quantum chemical simulations, and we correct for the systematic errors introduced by this approximation through the Gaussian process regression (see below). We note that previous work has shown that the value of Δ <i>E</i> <sub>electronic</sub> obtained for these types of redox reactions strongly correlates linearly with the full Δ <i>G</i> <sub>r</sub>° prediction obtained from including ro–vibrational enthalpic and entropic contribution (data not shown). </p> </div> <div class="transformingchemicalpotentialse°(majorspeciesatph=0)tobiochemicalpotentials,e′°(ph=7)" title="sec"> <div class="title" tagx="title" title="title">
Transforming Chemical Potentials <i>E</i>°(Major Species at pH = 0) to Biochemical Potentials, <i>E</i>′°(pH = 7) </div> <p>Having estimated the standard redox potential <i>E</i>°(major species at pH = 0), we use cheminformatic p <i>K</i> <sub>a</sub> estimates of reactants and the Alberty–Legendre transform <sup> <a href="#ref41">41</a> </sup> (p 67, eqs 4.4–10) to convert <i>E</i>°(major species at pH = 0) to the <i>transformed</i> standard redox potential, <i>E</i>′°(pH = 7), which is a function of pH. <sup> <a href="#ref63">63</a> </sup> To estimate the p <i>K</i> <sub>a</sub>’s of all reactants in every redox reaction of interest, we use the ChemAxon calculator plugin (Marvin 17.7.0, 2017, ChemAxon) <i>cxcalc pka</i>. Internally, the p <i>K</i> <sub>a</sub> calculator plugin is based on the calculation of the partial charge of atoms in the molecule. <sup> <a href="#ref64">64</a>, <a href="#ref65">65</a> </sup> </p> </div> <div class="gaussianprocesscalibrationusingreactionfingerprints" title="sec"> <div class="title" tagx="title" title="title">
Gaussian Process Calibration Using Reaction Fingerprints </div> <p>We calibrate biochemical redox potentials <i>E</i>′°(pH = 7) obtained from quantum chemical simulations against available experimental data to correct for systematic errors in our simulations and the cheminformatic p <i>K</i> <sub>a</sub> estimates. One simple strategy of calibration of energy values would be to use a two-parameter linear regression. We note that our group has successfully employed this in the context of compounds for redox flow battery applications. <sup> <a href="#ref66">66</a>− <a href="#ref71">71</a> </sup> Here, we make use of the information provided by the difference between the structural similarity of products and substrates and utilize a GP regression approach with reaction fingerprints to calibrate redox potential energies. For our calibrations with Gaussian process regression we used GPMol, a Python library based on GPflow, <sup> <a href="#ref72">72</a> </sup> which in turn is a package for building Gaussian process models using TensorFlow. </p> <p>GP regression relies on a similarity or distance metric between data points. To construct the notion of similarity between reactions, we utilize reaction fingerprints. <sup> <a href="#ref35">35</a> </sup> Several possible choices of kernel function and distance measure between molecules and reactions exist. Generally, the distance measures between molecules and reactions make use of fingerprint representations of compounds, which encode the structure of a molecule in a binary vector form. <sup> <a href="#ref73">73</a> </sup> Although several varieties of reaction fingerprints exist, we tested our approach on the Morgan2 and AP3 (atom pair) reaction FPs of Schneider et al. <sup> <a href="#ref35">35</a> </sup> and obtain them from the difference of fingerprint vectors of products and substrates. We find that in the context of Gaussian processes for biochemical redox potential predictions Morgan fingerprints outperform AP3 fingerprints by approximately 5–4 mV. The Gaussian processes’ predictive power comes from learning relationships between relative distances in feature space, so it might be the case that for distance-based ML algorithms Morgan fingerprints are better. However, we note that we did not incorporate the usage of any agent to the weighting scheme, so doing so might change the results. Additional performance metrics for comparison can be seen in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b00297/suppl_file/oc9b00297_si_001.pdf">Table S1</a>. </p> <p>One key ability of Morgan fingerprints is that we can invert the bits back to a corresponding molecular substructure to perform our structure–energy analysis. For each molecule, we generate a counted Morgan fingerprint, <sup> <a href="#ref73">73</a> </sup> a fixed-length counted vector indicating the absence (zeros) or counted presence (positive) of a particular graph–connectivity–environment. Each environment captures the local topological information on a molecule by mapping the local vicinity of connected atoms, along with their formal charges, type of chemical bond, and its position relative to a cyclic structure. Our Morgan fingerprints are constructed from 2048 length-vectors using a radius of 6 and are reduced to a lower dimension (∼200) using a gradient boosted tree. We find that the prediction accuracy did not improve significantly upon using Morgan fingerprints with a larger radius. When we consider the difference of two Morgan fingerprints, we are looking at the subtraction and addition of molecular environments between two molecules. We expect that similar molecular transformations will have similar changes in molecular environments. When these vectors are normalized, the inner product between two reaction fingerprints <i>x</i> <sub>1</sub> and <i>x</i> <sub>2</sub> is the cosine distance, <sup> <a href="#ref74">74</a> </sup> which is a measure of similarity between two vectors. This notion is adapted in the field of Gaussian processes to quantify the similarity between reactions in GP regression. </p> <p>Predictions obtained from GPs can be interpreted as weighted averages of the training data, where the weights are probabilistic in nature. GPs are distinct because of their associated covariance functions (i.e., the kernel). <sup> <a href="#ref33">33</a> </sup> The equation below shows that the kernel we employ is a mixture of a squared exponential kernel and a noise kernel, which increases the robustness of the model. </p><div class="disp-formula" title="disp-formula"> <div class="graphic_UNKNOWN" /> </div>Here, <i>K</i> is the kernel function, <i>σ</i> <sub>rxn</sub> is the variance of the reaction fingerprint kernel, <i>l</i> is the length scale parameter, and <i>σ</i> <sub>noise</sub> is the white noise variance parameter. <p /> </div> <div class="databaseofexperimentalredoxpotentialsforcalibrationwithgaussianprocessregression" title="sec"> <div class="title" tagx="title" title="title">
Database of Experimental Redox Potentials for Calibration with Gaussian Process Regression </div> <p>To perform the calibration, a data set of available experimental transformed standard redox potentials is used. This data set of redox half-reactions and their associated transformed standard reduction potentials <i>E</i>′°(pH = 7) was generated by Bar-Even et al., <sup> <a href="#ref4">4</a> </sup> by compiling experimental equilibrium constants (i.e., standard Gibbs reaction energies) from the NIST database of Thermodynamics of Enzyme-Catalyzed Reactions (TECRDB) <sup> <a href="#ref40">40</a> </sup> and the Robert Alberty database of biochemical compound Gibbs formation energies. <sup> <a href="#ref41">41</a> </sup> The data set consists of 57 redox reactions that reduce a carbonyl functional group into an alcohol functional group and 24 redox reactions that reduce a carbonyl functional group into an amine group. </p> <p>To ensure we are not overfitting in the process of calibrating with GP regression, we used leave-one-out cross-validation (LOOCV), where we train our model on all data points except one point and predict its value. This is done for all 81 data points, and reported predictions, accuracies, and resulting scatter plots always come from untrained data. It is also important to note that GPs are inherently robust to overfitting since the training procedure penalizes more complex models (higher-rank kernels) via an objective function. <sup> <a href="#ref32">32</a> </sup> </p> </div> <div class="selectingamodelchemistrywithlowcomputationalcostandhighpredictionaccuracy" title="sec"> <div class="title" tagx="title" title="title">
Selecting a Model Chemistry with Low Computational Cost and High Prediction Accuracy </div> <p>To obtain fast and accurate predictions of redox potentials using the GP regression calibrated quantum chemistry strategy described above, we tested several different quantum model chemistries. A model chemistry consists of a combination of a geometry optimization (GO) procedure, a method to calculate the single-point electronic energies (SPEs) of optimized molecular geometries, as well as other modeling considerations, such as the number of geometric conformations per compound, the water solvation model, and the p <i>K</i> <sub>a</sub> estimation strategy. </p> <p>We explored the following set of methods to obtain optimized geometric conformations: the Universal Force Field (UFF), <sup> <a href="#ref61">61</a> </sup> PM7, <sup> <a href="#ref34">34</a> </sup> DFTB, <sup> <a href="#ref75">75</a> </sup> HF3-c, <sup> <a href="#ref76">76</a> </sup> and PBEh-3c. <sup> <a href="#ref77">77</a> </sup> Additionally, to compute the electronic energies of the optimized structures through single-point energy (SPE) calculations, we considered the following approaches: DFTB, PM7, HF3-c, PBEh-3c, DSD-PBEP86/SVP, <sup> <a href="#ref78">78</a> </sup> and DLPNO–CCD(T)/SVP. <sup> <a href="#ref79">79</a> </sup> Given computational resource constraints, only a subset of all possible combinations of geometry optimization and SPE procedures were explored. </p> <p>To select a model chemistry from the exploration set, we compared their prediction accuracies—after the GP regression calibration step described above—when tested on the set of experimental redox potentials. We quantified prediction accuracy using three different metrics: Pearson correlation coefficient ( <i>r</i>), the coefficient of determination ( <i>R</i> <sup>2</sup>), and mean absolute error (MAE, in mV). Before applying the GP regression calibration step, the double-hybrid functional and linear-scaling couple-clustered methods DSD-PBEP86 and DLPNO–CCD(T) resulted in the highest accuracy. Due to the relatively small variation in prediction accuracy, we picked the cheapest method that still gave reasonable accuracy (MAE &amp;lt; 30 meV), which turned out to be the PM7 semiempirical method for both geometry optimizations and single-point electronic energies. It should be noted that our geometry is a single, UFF optimized conformation per molecule. For PBEh-3c and HF-3c we also considered utilizing multiple conformers obtained with the ETKDG force field <sup> <a href="#ref62">62</a> </sup> with significant improvement in accuracy over a single conformer (see <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b00297/suppl_file/oc9b00297_si_001.pdf">Table S1</a>). For other chemical applications (e.g., redox flow batteries) and molecular data sets, adequately accounting for multiple conformers might have an important influence on the accuracy of the calculations. A single missed hydrogen bond in one reacting partner, but not the other, could easily introduce large errors in the predicted redox potential. As such the adequacy of using just a single geometry should be verified. </p> <p>To test for statistically significant differences of the prediction accuracy summary statistics of different model chemistries (e.g., GCM versus GP-calibrated PM7), we performed a nonparametric bootstrap hypothesis test: we subsampled data points from the table of redox reactions with experimental data and computed the prediction accuracy summary statistics [Pearson <i>r</i>, <i>R</i> <sup>2</sup>, MAE, root-mean-square error (RMSE)] for each subsample. We then count the fraction of subsampled GCM data sets that result in summary statistics that are greater than or equal in value to the summary statistics of the model chemistry we are comparing against (e.g., GP-calibrated PM7). </p> </div> <div class="groupcontributionmethod(gcm)estimatesofredoxpotentials" title="sec"> <div class="title" tagx="title" title="title">
Group Contribution Method (GCM) Estimates of Redox Potentials </div> <p>We compared the accuracies obtained using the GP regression calibrated quantum chemistry approach described above against those obtained with the commonly used GCM. <sup> <a href="#ref14">14</a>, <a href="#ref18">18</a>, <a href="#ref80">80</a> </sup> We use the GCM implementation of Noor et al. which was adapted to the thermodynamics of biochemical reactions. <sup> <a href="#ref18">18</a> </sup> Briefly, group energies of all compounds in the KEGG database are stored in a group matrix, with rows corresponding to compounds and columns corresponding to groups. Associated to each group is a group energy corresponding to the energy for the group’s major protonation state (major species) at pH = 7. To obtain a GCM-based estimate for a standard transformed reduction potential for the major species at pH = 7, we take the difference in group energy vectors for all products and substrate in the reaction. We then transform the resulting standard redox potentials <i>E</i>°(major species at pH = 7) to the standard <i>transformed</i> redox potential at pH = 7, <i>E</i>′°(pH = 7), using the same p <i>K</i> <sub>a</sub> estimates and Alberty transform approach described above. </p> </div> <div class="redoxnetworkexpansionalgorithmwithcheminformaticreactionstrings" title="sec"> <div class="title" tagx="title" title="title">
Redox Network Expansion Algorithm with Cheminformatic Reaction Strings </div> <p>We implemented a network expansion algorithm to iteratively reduce carbonyl functional groups and oxidized alcohol/amine functional groups for a subset of the natural metabolites of the KEGG database. Due to constraints in available computational resources to run electronic structure calculations, we used the set of all metabolic compounds in the KEGG database with 21 carbon atoms or fewer as seed metabolites. Using the RDKit open source cheminformatics software, we identify carbonyl functional groups (alcohol/amine functional groups) in each of the seed KEGG metabolites and transform them through a redox reaction to the corresponding reduced (oxidized) alcohol/amine (carbonyl). In practice, this is done using SMARTS reaction strings as implemented in RDKit and iteratively applying them to SMILES representations <sup> <a href="#ref43">43</a> </sup> of the KEGG metabolites. The SMARTS reaction strings for the reduction and oxidation reactions considered here are shown in <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b00297/suppl_file/oc9b00297_si_001.pdf">Tables S2 and S3</a>. </p> <p>We iteratively apply the reduction/oxidation reaction transformations to the compounds generated from KEGG seed metabolites. We terminate the iterations when the resulting product has, in the case of reductions, no more carbonyl functional groups that can be reduced (or in the case of oxidative transformations, no more alcohol/amine functional groups that can be oxidized to carbonyls). All software was written using the Python programming language. </p> </div> </div> </div> <div class="back" title="back"> <div class="title" tagx="title" title="title">
Supporting Information Available</div> <p>The Supporting Information is available free of charge on the <a href="http://pubs.acs.org">ACS Publications website</a> at DOI: <a href="http://pubs.acs.org/doi/abs/10.1021/acscentsci.9b00297">10.1021/acscentsci.9b00297</a>. </p><ul> <li> <p>Additional data, including model chemistry prediction accuracies and cheminformatic SMARTS reaction strings used to generate redox network ( <a href="http://pubs.acs.org/doi/suppl/10.1021/acscentsci.9b00297/suppl_file/oc9b00297_si_001.pdf">PDF</a>) </p> </li> </ul> <p /> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="oc9b00297_si_001.pdf">LINK</a>  <p>oc9b00297_si_001.pdf</p>  </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p> <sup>#</sup> A.J. and B.S.-L. contributed equally to this work </p> <p>The authors declare no competing financial interest. </p> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>We thank members of the Aspuru-Guzik lab for comments on the manuscript. The authors thank Harvard Research Computing for their support on using the Odyssey cluster. A.A.-G., A.J., and B.S.-L. acknowledge support from SEAS^NVIDIA, Massively Parallel Programming and Computing (332986). A.A.-G. acknowledges the generous support of Anders G. Froseth for this program and the Canada 150 Research Chairs Program. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="ref1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nelson</span> <span class="given-names" tagx="given-names" title="given-names">D. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lehninger</span> <span class="given-names" tagx="given-names" title="given-names">A. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cox</span> <span class="given-names" tagx="given-names" title="given-names">M. M.</span> </span> </span> <span class="source" tagx="source" title="source">Lehninger Principles of Biochemistry </span>; <span class="publisher-name" tagx="publisher-name" title="publisher-name">Macmillan</span>, <span class="year" tagx="year" title="year">2008</span>. </span> </li> <li tag="ref"> <a name="ref2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar-Even</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flamholz</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reznik</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liebermeister</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milo</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Pathway Thermodynamics Highlights Kinetic Obstacles in Central Metabolism </span>. <span class="source" tagx="source" title="source">PLoS Comput. Biol.</span> <span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">10</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003483</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pcbi.1003483">10.1371/journal.pcbi.1003483</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24586134">24586134</a> </span> </span> </li> <li tag="ref"> <a name="ref3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henry</span> <span class="given-names" tagx="given-names" title="given-names">C. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Broadbelt</span> <span class="given-names" tagx="given-names" title="given-names">L. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hatzimanikatis</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Thermodynamics-Based Metabolic Flux Analysis </span>. <span class="source" tagx="source" title="source">Biophys. J.</span> <span class="year" tagx="year" title="year">2007</span>, <span class="volume" tagx="volume" title="volume">92</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">1792</span>– <span class="lpage" tagx="lpage" title="lpage">1805</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1529/biophysj.106.093138">10.1529/biophysj.106.093138</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17172310">17172310</a> </span> </span> </li> <li tag="ref"> <a name="ref4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar-Even</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flamholz</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milo</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Thermodynamic Constraints Shape the Structure of Carbon Fixation Pathways </span>. <span class="source" tagx="source" title="source">Biochim. Biophys. Acta, Bioenerg.</span> <span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">1817</span> <span class="issue" tagx="issue" title="issue">9</span> <span class="fpage" tagx="fpage" title="fpage">1646</span>– <span class="lpage" tagx="lpage" title="lpage">1659</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bbabio.2012.05.002">10.1016/j.bbabio.2012.05.002</a> </span>. </span> </li> <li tag="ref"> <a name="ref5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krumholz</span> <span class="given-names" tagx="given-names" title="given-names">E. W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Libourel</span> <span class="given-names" tagx="given-names" title="given-names">I. G. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Thermodynamic Constraints Improve Metabolic Networks </span>. <span class="source" tagx="source" title="source">Biophys. J. </span> <span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">113</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">679</span>– <span class="lpage" tagx="lpage" title="lpage">689</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bpj.2017.06.018">10.1016/j.bpj.2017.06.018</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28793222">28793222</a> </span> </span> </li> <li tag="ref"> <a name="ref6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krebs</span> <span class="given-names" tagx="given-names" title="given-names">H. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kornberg</span> <span class="given-names" tagx="given-names" title="given-names">H. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burton</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A Survey of the Energy Transformations in Living Matter </span>. <span class="source" tagx="source" title="source">Rev. Physiol., Biochem. Pharmacol.</span> <span class="year" tagx="year" title="year">1957</span>, <span class="volume" tagx="volume" title="volume">49</span>, <span class="fpage" tagx="fpage" title="fpage">212</span>– <span class="lpage" tagx="lpage" title="lpage">298</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/BF02269485">10.1007/BF02269485</a> </span>. </span> </li> <li tag="ref"> <a name="ref7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alberty</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A Short History of the Thermodynamics of Enzyme-Catalyzed Reactions </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">2004</span>, <span class="volume" tagx="volume" title="volume">279</span> <span class="issue" tagx="issue" title="issue">27</span> <span class="fpage" tagx="fpage" title="fpage">27831</span>– <span class="lpage" tagx="lpage" title="lpage">27836</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1074/jbc.X400003200">10.1074/jbc.X400003200</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15073189">15073189</a> </span> </span> </li> <li tag="ref"> <a name="ref8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burton</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krebs</span> <span class="given-names" tagx="given-names" title="given-names">H. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The Free-Energy Changes Associated with the Individual Steps of the Tricarboxylic Acid Cycle, Glycolysis and Alcoholic Fermentation and with the Hydrolysis of the Pyrophosphate Groups of Adenosinetriphosphate </span>. <span class="source" tagx="source" title="source">Biochem. J.</span> <span class="year" tagx="year" title="year">1953</span>, <span class="volume" tagx="volume" title="volume">54</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">94</span>– <span class="lpage" tagx="lpage" title="lpage">107</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1042/bj0540094">10.1042/bj0540094</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/13058838">13058838</a> </span> </span> </li> <li tag="ref"> <a name="ref9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kiparissides</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hatzimanikatis</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Thermodynamics-Based Metabolite Sensitivity Analysis in Metabolic Networks </span>. <span class="source" tagx="source" title="source">Metab. Eng.</span> <span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">39</span>, <span class="fpage" tagx="fpage" title="fpage">117</span>– <span class="lpage" tagx="lpage" title="lpage">127</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ymben.2016.11.006">10.1016/j.ymben.2016.11.006</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27845184">27845184</a> </span> </span> </li> <li tag="ref"> <a name="ref10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar-Even</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lewis</span> <span class="given-names" tagx="given-names" title="given-names">N. E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milo</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Design and Analysis of Synthetic Carbon Fixation Pathways </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A. </span> <span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">107</span> <span class="issue" tagx="issue" title="issue">19</span> <span class="fpage" tagx="fpage" title="fpage">8889</span>– <span class="lpage" tagx="lpage" title="lpage">8894</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0907176107">10.1073/pnas.0907176107</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20410460">20410460</a> </span> </span> </li> <li tag="ref"> <a name="ref11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flamholz</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar-Even</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liebermeister</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milo</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Glycolytic Strategy as a Tradeoff between Energy Yield and Protein Cost </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span> <span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">110</span> <span class="issue" tagx="issue" title="issue">24</span> <span class="fpage" tagx="fpage" title="fpage">10039</span>– <span class="lpage" tagx="lpage" title="lpage">10044</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1215283110">10.1073/pnas.1215283110</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23630264">23630264</a> </span> </span> </li> <li tag="ref"> <a name="ref12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hadadi</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hafner</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shajkofci</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zisaki</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hatzimanikatis</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="mixed-article-title" title="mixed-article-title">ATLAS of Biochemistry: A Repository of All Possible Biochemical Reactions for Synthetic Biology and Metabolic Engineering Studies </span>. <span class="source" tagx="source" title="source">ACS Synth. Biol. </span> <span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">5</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="fpage" tagx="fpage" title="fpage">1155</span>– <span class="lpage" tagx="lpage" title="lpage">1166</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acssynbio.6b00054">10.1021/acssynbio.6b00054</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27404214">27404214</a> </span> </span> </li> <li tag="ref"> <a name="ref13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Siegel</span> <span class="given-names" tagx="given-names" title="given-names">J. B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">A. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poust</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wargacki</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar-Even</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Louw</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">B. W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eiben</span> <span class="given-names" tagx="given-names" title="given-names">C. B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tran</span> <span class="given-names" tagx="given-names" title="given-names">H. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; et al. <span class="mixed-article-title" title="mixed-article-title">Computational Protein Design Enables a Novel One-Carbon Assimilation Pathway </span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A. </span> <span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">112</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">3704</span>– <span class="lpage" tagx="lpage" title="lpage">3709</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1500545112">10.1073/pnas.1500545112</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25775555">25775555</a> </span> </span> </li> <li tag="ref"> <a name="ref14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benson</span> <span class="given-names" tagx="given-names" title="given-names">S. W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buss</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Additivity Rules for the Estimation of Molecular Properties. Thermodynamic Properties </span>. <span class="source" tagx="source" title="source">J. Chem. Phys.</span> <span class="year" tagx="year" title="year">1958</span>, <span class="volume" tagx="volume" title="volume">29</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">546</span>– <span class="lpage" tagx="lpage" title="lpage">572</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1063/1.1744539">10.1063/1.1744539</a> </span>. </span> </li> <li tag="ref"> <a name="ref15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mavrovouniotis</span> <span class="given-names" tagx="given-names" title="given-names">M. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Group Contributions for Estimating Standard Gibbs Energies of Formation of Biochemical Compounds in Aqueous Solution </span>. <span class="source" tagx="source" title="source">Biotechnol. Bioeng.</span> <span class="year" tagx="year" title="year">1990</span>, <span class="volume" tagx="volume" title="volume">36</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="fpage" tagx="fpage" title="fpage">1070</span>– <span class="lpage" tagx="lpage" title="lpage">1082</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bit.260361013">10.1002/bit.260361013</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18595046">18595046</a> </span> </span> </li> <li tag="ref"> <a name="ref16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mavrovouniotis</span> <span class="given-names" tagx="given-names" title="given-names">M. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Estimation of Standard Gibbs Energy Changes of Biotransformations </span>. <span class="source" tagx="source" title="source">J. Biol. Chem.</span> <span class="year" tagx="year" title="year">1991</span>, <span class="volume" tagx="volume" title="volume">266</span> <span class="issue" tagx="issue" title="issue">22</span> <span class="fpage" tagx="fpage" title="fpage">14440</span>– <span class="lpage" tagx="lpage" title="lpage">14445</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1860851">1860851</a> </span> </span> </li> <li tag="ref"> <a name="ref17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jankowski</span> <span class="given-names" tagx="given-names" title="given-names">M. D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henry</span> <span class="given-names" tagx="given-names" title="given-names">C. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Broadbelt</span> <span class="given-names" tagx="given-names" title="given-names">L. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hatzimanikatis</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Group Contribution Method for Thermodynamic Analysis of Complex Metabolic Networks </span>. <span class="source" tagx="source" title="source">Biophys. J.</span> <span class="year" tagx="year" title="year">2008</span>, <span class="volume" tagx="volume" title="volume">95</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">1487</span>– <span class="lpage" tagx="lpage" title="lpage">1499</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1529/biophysj.107.124784">10.1529/biophysj.107.124784</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18645197">18645197</a> </span> </span> </li> <li tag="ref"> <a name="ref18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haraldsdóttir</span> <span class="given-names" tagx="given-names" title="given-names">H. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milo</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fleming</span> <span class="given-names" tagx="given-names" title="given-names">R. M. T.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Consistent Estimation of Gibbs Energy Using Component Contributions </span>. <span class="source" tagx="source" title="source">PLoS Comput. Biol.</span> <span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">9</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003098</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pcbi.1003098">10.1371/journal.pcbi.1003098</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23874165">23874165</a> </span> </span> </li> <li tag="ref"> <a name="ref19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar-Even</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flamholz</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lubling</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davidi</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milo</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">An Integrated Open Framework for Thermodynamics of Reactions That Combines Accuracy and Coverage </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">28</span> <span class="issue" tagx="issue" title="issue">15</span> <span class="fpage" tagx="fpage" title="fpage">2037</span>– <span class="lpage" tagx="lpage" title="lpage">2044</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bts317">10.1093/bioinformatics/bts317</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22645166">22645166</a> </span> </span> </li> <li tag="ref"> <a name="ref20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flamholz</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar-Even</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Milo</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">eQuilibrator--the Biochemical Thermodynamics Calculator </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">40</span> <span class="issue" tagx="issue" title="issue">D1</span> <span class="fpage" tagx="fpage" title="fpage">D770</span>– <span class="lpage" tagx="lpage" title="lpage">D775</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkr874">10.1093/nar/gkr874</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22064852">22064852</a> </span> </span> </li> <li tag="ref"> <a name="ref21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jinich</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rappoport</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dunn</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanchez-Lengeling</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Olivares-Amaya</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Even</span> <span class="given-names" tagx="given-names" title="given-names">A. B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantum Chemical Approach to Estimating the Thermodynamics of Metabolic Reactions </span>. <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">7022</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep07022">10.1038/srep07022</a> </span>. </span> </li> <li tag="ref"> <a name="ref22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jinich</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Flamholz</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S.-J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanchez-Lengeling</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cotton</span> <span class="given-names" tagx="given-names" title="given-names">C. A. R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bar-Even</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantum Chemistry Reveals Thermodynamic Principles of Redox Biochemistry </span>. <span class="source" tagx="source" title="source">PLoS Comput. Biol.</span> <span class="year" tagx="year" title="year">2018</span>, <span class="volume" tagx="volume" title="volume">14</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">e1006471</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pcbi.1006471">10.1371/journal.pcbi.1006471</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/30356318">30356318</a> </span> </span> </li> <li tag="ref"> <a name="ref23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Banerjee</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="source" tagx="source" title="source">Redox Biochemistry</span>; <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley &amp;amp; Sons</span>, <span class="year" tagx="year" title="year">2007</span>. </span> </li> <li tag="ref"> <a name="ref24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hadadi</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ataman</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hatzimanikatis</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panayiotou</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular Thermodynamics of Metabolism: Quantum Thermochemical Calculations for Key Metabolites </span>. <span class="source" tagx="source" title="source">Phys. Chem. Chem. Phys.</span> <span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">17</span> <span class="issue" tagx="issue" title="issue">16</span> <span class="fpage" tagx="fpage" title="fpage">10438</span>– <span class="lpage" tagx="lpage" title="lpage">10453</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/C4CP05825A">10.1039/C4CP05825A</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25799954">25799954</a> </span> </span> </li> <li tag="ref"> <a name="ref25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jinich</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanchez-Lengeling</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goldford</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanders</span> <span class="given-names" tagx="given-names" title="given-names">J. N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Segrè</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A Thermodynamic Atlas of Carbon Redox Chemical Space </span>. <span class="year" tagx="year" title="year">2019</span>, bioRxiv245811. <span class="uri" title="uri"> <a href="https://doi.org/10.1101/245811">https://doi.org/10.1101/245811</a> </span>. </span> </li> <li tag="ref"> <a name="ref26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hansen</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Biegler</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramakrishnan</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pronobis</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">von Lilienfeld</span> <span class="given-names" tagx="given-names" title="given-names">O. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Müller</span> <span class="given-names" tagx="given-names" title="given-names">K.-R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tkatchenko</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Machine Learning Predictions of Molecular Properties: Accurate Many-Body Potentials and Nonlocality in Chemical Space </span>. <span class="source" tagx="source" title="source">J. Phys. Chem. Lett.</span> <span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">2326</span>– <span class="lpage" tagx="lpage" title="lpage">2331</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acs.jpclett.5b00831">10.1021/acs.jpclett.5b00831</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26113956">26113956</a> </span> </span> </li> <li tag="ref"> <a name="ref27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramakrishnan</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dral</span> <span class="given-names" tagx="given-names" title="given-names">P. O.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rupp</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">von Lilienfeld</span> <span class="given-names" tagx="given-names" title="given-names">O. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Big Data Meets Quantum Chemistry Approximations: The Δ-Machine Learning Approach </span>. <span class="source" tagx="source" title="source">J. Chem. Theory Comput.</span> <span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">11</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">2087</span>– <span class="lpage" tagx="lpage" title="lpage">2096</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acs.jctc.5b00099">10.1021/acs.jctc.5b00099</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26574412">26574412</a> </span> </span> </li> <li tag="ref"> <a name="ref28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rupp</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tkatchenko</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Müller</span> <span class="given-names" tagx="given-names" title="given-names">K.-R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">von Lilienfeld</span> <span class="given-names" tagx="given-names" title="given-names">O. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Fast and Accurate Modeling of Molecular Atomization Energies with Machine Learning </span>. <span class="source" tagx="source" title="source">Phys. Rev. Lett.</span> <span class="year" tagx="year" title="year">2012</span>, <span class="volume" tagx="volume" title="volume">108</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">058301</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1103/PhysRevLett.108.058301">10.1103/PhysRevLett.108.058301</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22400967">22400967</a> </span> </span> </li> <li tag="ref"> <a name="ref29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Isayev</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roitberg</span> <span class="given-names" tagx="given-names" title="given-names">A. E.</span> </span> <span class="mixed-article-title" title="mixed-article-title">ANI-1: An Extensible Neural Network Potential with DFT Accuracy at Force Field Computational Cost </span>. <span class="source" tagx="source" title="source">Chem. Sci.</span> <span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">8</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">3192</span>– <span class="lpage" tagx="lpage" title="lpage">3203</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/C6SC05720A">10.1039/C6SC05720A</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28507695">28507695</a> </span> </span> </li> <li tag="ref"> <a name="ref30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pyzer-Knapp</span> <span class="given-names" tagx="given-names" title="given-names">E. O.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simm</span> <span class="given-names" tagx="given-names" title="given-names">G. N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A Bayesian Approach to Calibrating High-Throughput Virtual Screening Results and Application to Organic Photovoltaic Materials </span>. <span class="source" tagx="source" title="source">Mater. Horiz.</span> <span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">226</span>– <span class="lpage" tagx="lpage" title="lpage">233</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/C5MH00282F">10.1039/C5MH00282F</a> </span>. </span> </li> <li tag="ref"> <a name="ref31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lopez</span> <span class="given-names" tagx="given-names" title="given-names">S. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanchez-Lengeling</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Goes Soares</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Design Principles and Top Non-Fullerene Acceptor Candidates for Organic Photovoltaics </span>. <span class="source" tagx="source" title="source">Joule</span> <span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">1</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">857</span>– <span class="lpage" tagx="lpage" title="lpage">870</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.joule.2017.10.006">10.1016/j.joule.2017.10.006</a> </span>. </span> </li> <li tag="ref"> <a name="ref32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rasmussen</span> <span class="given-names" tagx="given-names" title="given-names">C. E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">C. K. I.</span> </span> </span> <span class="source" tagx="source" title="source">Gaussian Processes for Machine Learning </span>; <span class="publisher-name" tagx="publisher-name" title="publisher-name">MIT Press</span>, <span class="year" tagx="year" title="year">2006</span>. </span> </li> <li tag="ref"> <a name="ref33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kung</span> <span class="given-names" tagx="given-names" title="given-names">S. Y.</span> </span> </span> <span class="source" tagx="source" title="source">Kernel Methods and Machine Learning </span>; <span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span>, <span class="year" tagx="year" title="year">2014</span>. </span> </li> <li tag="ref"> <a name="ref34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stewart</span> <span class="given-names" tagx="given-names" title="given-names">J. J. P. </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optimization of Parameters for Semiempirical Methods VI: More Modifications to the NDDO Approximations and Re-Optimization of Parameters </span>. <span class="source" tagx="source" title="source">J. Mol. Model.</span> <span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">19</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">32</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00894-012-1667-x">10.1007/s00894-012-1667-x</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23187683">23187683</a> </span> </span> </li> <li tag="ref"> <a name="ref35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneider</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lowe</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sayle</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Landrum</span> <span class="given-names" tagx="given-names" title="given-names">G. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Development of a Novel Fingerprint for Chemical Reactions and Its Application to Large-Scale Reaction Classification and Similarity </span>. <span class="source" tagx="source" title="source">J. Chem. Inf. Model. </span> <span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">55</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">39</span>– <span class="lpage" tagx="lpage" title="lpage">53</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci5006614">10.1021/ci5006614</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25541888">25541888</a> </span> </span> </li> <li tag="ref"> <a name="ref36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goto</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hattori</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aoki-Kinoshita</span> <span class="given-names" tagx="given-names" title="given-names">K. F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Itoh</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawashima</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Katayama</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Araki</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirakawa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">From Genomics to Chemical Genomics: New Developments in KEGG </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2006</span>, <span class="volume" tagx="volume" title="volume">34</span> <span class="issue" tagx="issue" title="issue">90001</span> <span class="fpage" tagx="fpage" title="fpage">D354</span>– <span class="lpage" tagx="lpage" title="lpage">D357</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkj102">10.1093/nar/gkj102</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16381885">16381885</a> </span> </span> </li> <li tag="ref"> <a name="ref37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Furumichi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanabe</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morishima</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">KEGG: New Perspectives on Genomes, Pathways, Diseases and Drugs </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">45</span> <span class="issue" tagx="issue" title="issue">D1</span> <span class="fpage" tagx="fpage" title="fpage">D353</span>– <span class="lpage" tagx="lpage" title="lpage">D361</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkw1092">10.1093/nar/gkw1092</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27899662">27899662</a> </span> </span> </li> <li tag="ref"> <a name="ref38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kawashima</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Furumichi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanabe</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">KEGG as a Reference Resource for Gene and Protein Annotation </span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">44</span> <span class="issue" tagx="issue" title="issue">D1</span> <span class="fpage" tagx="fpage" title="fpage">D457</span>– <span class="lpage" tagx="lpage" title="lpage">D462</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkv1070">10.1093/nar/gkv1070</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26476454">26476454</a> </span> </span> </li> <li tag="ref"> <a name="ref39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goto</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">KEGG Kyoto Encyclopedia of Genes and Genomes</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span> <span class="year" tagx="year" title="year">2000</span>, <span class="volume" tagx="volume" title="volume">28</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">27</span>– <span class="lpage" tagx="lpage" title="lpage">30</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/28.1.27">10.1093/nar/28.1.27</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173">10592173</a> </span> </span> </li> <li tag="ref"> <a name="ref40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goldberg</span> <span class="given-names" tagx="given-names" title="given-names">R. N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tewari</span> <span class="given-names" tagx="given-names" title="given-names">Y. B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhat</span> <span class="given-names" tagx="given-names" title="given-names">T. N.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Thermodynamics of Enzyme-Catalyzed Reactions--a Database for Quantitative Biochemistry </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="year" tagx="year" title="year">2004</span>, <span class="volume" tagx="volume" title="volume">20</span> <span class="issue" tagx="issue" title="issue">16</span> <span class="fpage" tagx="fpage" title="fpage">2874</span>– <span class="lpage" tagx="lpage" title="lpage">2877</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/bth314">10.1093/bioinformatics/bth314</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15145806">15145806</a> </span> </span> </li> <li tag="ref"> <a name="ref41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alberty</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span> </span> <span class="source" tagx="source" title="source">Thermodynamics of Biochemical Reactions </span>; <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley &amp;amp; Sons</span>, <span class="year" tagx="year" title="year">2005</span>. </span> </li> <li tag="ref"> <a name="ref42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klamt</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schüürmann</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="mixed-article-title" title="mixed-article-title">COSMO: A New Approach to Dielectric Screening in Solvents with Explicit Expressions for the Screening Energy and Its Gradient </span>. <span class="source" tagx="source" title="source">J. Chem. Soc., Perkin Trans. 2 </span> <span class="year" tagx="year" title="year">1993</span>, <span class="volume" tagx="volume" title="volume">0</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">799</span>– <span class="lpage" tagx="lpage" title="lpage">805</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/P29930000799">10.1039/P29930000799</a> </span>. </span> </li> <li tag="ref"> <a name="ref43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weininger</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="mixed-article-title" title="mixed-article-title">SMILES, a Chemical Language and Information System. 1. Introduction to Methodology and Encoding Rules </span>. <span class="source" tagx="source" title="source">J. Chem. Inf. Model.</span> <span class="year" tagx="year" title="year">1988</span>, <span class="volume" tagx="volume" title="volume">28</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">31</span>– <span class="lpage" tagx="lpage" title="lpage">36</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci00057a005">10.1021/ci00057a005</a> </span>. </span> </li> <li tag="ref"> <a name="ref44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heijenoort</span> <span class="given-names" tagx="given-names" title="given-names">J. v.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Formation of the Glycan Chains in the Synthesis of Bacterial Peptidoglycan </span>. <span class="source" tagx="source" title="source">Glycobiology</span> <span class="year" tagx="year" title="year">2001</span>, <span class="volume" tagx="volume" title="volume">11</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">25R</span>– <span class="lpage" tagx="lpage" title="lpage">36R</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/glycob/11.3.25R">10.1093/glycob/11.3.25R</a> </span>. </span> </li> <li tag="ref"> <a name="ref45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Brien</span> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Siraki</span> <span class="given-names" tagx="given-names" title="given-names">A. G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shangari</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Aldehyde Sources, Metabolism, Molecular Toxicity Mechanisms, and Possible Effects on Human Health </span>. <span class="source" tagx="source" title="source">Crit. Rev. Toxicol.</span> <span class="year" tagx="year" title="year">2005</span>, <span class="volume" tagx="volume" title="volume">35</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">609</span>– <span class="lpage" tagx="lpage" title="lpage">662</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/10408440591002183">10.1080/10408440591002183</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16417045">16417045</a> </span> </span> </li> <li tag="ref"> <a name="ref46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferguson</span> <span class="given-names" tagx="given-names" title="given-names">G. P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Protective Mechanisms against Toxic Electrophiles in Escherichia Coli </span>. <span class="source" tagx="source" title="source">Trends Microbiol.</span> <span class="year" tagx="year" title="year">1999</span>, <span class="volume" tagx="volume" title="volume">7</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">242</span>– <span class="lpage" tagx="lpage" title="lpage">247</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0966-842X(99)01510-3">10.1016/S0966-842X(99)01510-3</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10366861">10366861</a> </span> </span> </li> <li tag="ref"> <a name="ref47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salimbeni</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deisenroth</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="source" tagx="source" title="source">Doubly Stochastic Variational Inference for Deep Gaussian Processes </span>, 31st Conference on Neural Information Processing Systems (NIPS 2017), <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Long Beach, CA</span>, <span class="year" tagx="year" title="year">2017</span>; pp <span class="fpage" tagx="fpage" title="fpage">4588</span>– <span class="lpage" tagx="lpage" title="lpage">4599</span>. </span> </li> <li tag="ref"> <a name="ref48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gal</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van der Wilk</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rasmussen</span> <span class="given-names" tagx="given-names" title="given-names">C. E.</span> </span> </span> <span class="source" tagx="source" title="source">Distributed Variational Inference in Sparse Gaussian Process Regression and Latent Variable Models </span>, Proceedings of Neural Information Processing Systems <span class="year" tagx="year" title="year">2014</span>; pp <span class="fpage" tagx="fpage" title="fpage">3257</span>– <span class="lpage" tagx="lpage" title="lpage">3265</span>. </span> </li> <li tag="ref"> <a name="ref49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herr</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Parkhill</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The Many-Body Expansion Combined with Neural Networks </span>. <span class="source" tagx="source" title="source">J. Chem. Phys.</span> <span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">146</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">014106</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1063/1.4973380">10.1063/1.4973380</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28063436">28063436</a> </span> </span> </li> <li tag="ref"> <a name="ref50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramsundar</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feinberg</span> <span class="given-names" tagx="given-names" title="given-names">E. N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gomes</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Geniesse</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pappu</span> <span class="given-names" tagx="given-names" title="given-names">A. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leswing</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pande</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="mixed-article-title" title="mixed-article-title">MoleculeNet A Benchmark for Molecular Machine Learning </span>. <span class="source" tagx="source" title="source">Chem. Sci.</span> <span class="year" tagx="year" title="year">2018</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">513</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/C7SC02664A">10.1039/C7SC02664A</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29629118">29629118</a> </span> </span> </li> <li tag="ref"> <a name="ref51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Faber</span> <span class="given-names" tagx="given-names" title="given-names">F. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hutchison</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gilmer</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schoenholz</span> <span class="given-names" tagx="given-names" title="given-names">S. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dahl</span> <span class="given-names" tagx="given-names" title="given-names">G. E.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vinyals</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kearnes</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riley</span> <span class="given-names" tagx="given-names" title="given-names">P. F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">von Lilienfeld</span> <span class="given-names" tagx="given-names" title="given-names">O. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Prediction Errors of Molecular Machine Learning Models Lower than Hybrid DFT Error </span>. <span class="source" tagx="source" title="source">J. Chem. Theory Comput. </span> <span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">13</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">5255</span>– <span class="lpage" tagx="lpage" title="lpage">5264</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acs.jctc.7b00577">10.1021/acs.jctc.7b00577</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28926232">28926232</a> </span> </span> </li> <li tag="ref"> <a name="ref52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raymond</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Segrè</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The Effect of Oxygen on Biochemical Networks and the Evolution of Complex Life </span>. <span class="source" tagx="source" title="source">Science</span> <span class="year" tagx="year" title="year">2006</span>, <span class="volume" tagx="volume" title="volume">311</span> <span class="issue" tagx="issue" title="issue">5768</span> <span class="fpage" tagx="fpage" title="fpage">1764</span>– <span class="lpage" tagx="lpage" title="lpage">1767</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1118439">10.1126/science.1118439</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16556842">16556842</a> </span> </span> </li> <li tag="ref"> <a name="ref53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zubarev</span> <span class="given-names" tagx="given-names" title="given-names">D. Y.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rappoport</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Uncertainty of Prebiotic Scenarios: The Case of the Non-Enzymatic Reverse Tricarboxylic Acid Cycle </span>. <span class="source" tagx="source" title="source">Sci. Rep.</span> <span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">8009</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep08009">10.1038/srep08009</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25620471">25620471</a> </span> </span> </li> <li tag="ref"> <a name="ref54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riniker</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Landrum</span> <span class="given-names" tagx="given-names" title="given-names">G. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Similarity Maps - a Visualization Strategy for Molecular Fingerprints and Machine-Learning Methods </span>. <span class="source" tagx="source" title="source">J. Cheminf. </span> <span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">5</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">43</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1758-2946-5-43">10.1186/1758-2946-5-43</a> </span>. </span> </li> <li tag="ref"> <a name="ref55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Asami</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakano</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujioka</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Plant Brassinosteroid Hormones</span>. <span class="source" tagx="source" title="source">Vitam. Horm.</span> <span class="year" tagx="year" title="year">2005</span>, <span class="volume" tagx="volume" title="volume">72</span>, <span class="fpage" tagx="fpage" title="fpage">479</span>– <span class="lpage" tagx="lpage" title="lpage">504</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0083-6729(05)72014-8">10.1016/S0083-6729(05)72014-8</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16492480">16492480</a> </span> </span> </li> <li tag="ref"> <a name="ref56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujioka</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yokota</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Biosynthesis and Metabolism of Brassinosteroids</span>. <span class="source" tagx="source" title="source">Annu. Rev. Plant Biol.</span> <span class="year" tagx="year" title="year">2003</span>, <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">137</span>– <span class="lpage" tagx="lpage" title="lpage">164</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.arplant.54.031902.134921">10.1146/annurev.arplant.54.031902.134921</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14502988">14502988</a> </span> </span> </li> <li tag="ref"> <a name="ref57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mosblech</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feussner</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heilmann</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Oxylipins: Structurally Diverse Metabolites from Fatty Acid Oxidation </span>. <span class="source" tagx="source" title="source">Plant Physiol. Biochem.</span> <span class="year" tagx="year" title="year">2009</span>, <span class="volume" tagx="volume" title="volume">47</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">511</span>– <span class="lpage" tagx="lpage" title="lpage">517</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.plaphy.2008.12.011">10.1016/j.plaphy.2008.12.011</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19167233">19167233</a> </span> </span> </li> <li tag="ref"> <a name="ref58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zanger</span> <span class="given-names" tagx="given-names" title="given-names">U. M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwab</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Cytochrome P450 Enzymes in Drug Metabolism: Regulation of Gene Expression, Enzyme Activities, and Impact of Genetic Variation </span>. <span class="source" tagx="source" title="source">Pharmacol. Ther.</span> <span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">138</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">103</span>– <span class="lpage" tagx="lpage" title="lpage">141</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.pharmthera.2012.12.007">10.1016/j.pharmthera.2012.12.007</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23333322">23333322</a> </span> </span> </li> <li tag="ref"> <a name="ref59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tschumper</span> <span class="given-names" tagx="given-names" title="given-names">G. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaefer</span> <span class="given-names" tagx="given-names" title="given-names">H. F.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Predicting Electron Affinities with Density Functional Theory: Some Positive Results for Negative Ions </span>. <span class="source" tagx="source" title="source">J. Chem. Phys.</span> <span class="year" tagx="year" title="year">1997</span>, <span class="volume" tagx="volume" title="volume">107</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">2529</span>– <span class="lpage" tagx="lpage" title="lpage">2541</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1063/1.474593">10.1063/1.474593</a> </span>. </span> </li> <li tag="ref"> <a name="ref60" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simons</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular Anions </span>. <span class="source" tagx="source" title="source">J. Phys. Chem. A</span> <span class="year" tagx="year" title="year">2008</span>, <span class="volume" tagx="volume" title="volume">112</span> <span class="issue" tagx="issue" title="issue">29</span> <span class="fpage" tagx="fpage" title="fpage">6401</span>– <span class="lpage" tagx="lpage" title="lpage">6511</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/jp711490b">10.1021/jp711490b</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18630888">18630888</a> </span> </span> </li> <li tag="ref"> <a name="ref61" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rappe</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Casewit</span> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Colwell</span> <span class="given-names" tagx="given-names" title="given-names">K. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goddard</span> <span class="given-names" tagx="given-names" title="given-names">W. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skiff</span> <span class="given-names" tagx="given-names" title="given-names">W. M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">UFF, a Full Periodic Table Force Field for Molecular Mechanics and Molecular Dynamics Simulations </span>. <span class="source" tagx="source" title="source">J. Am. Chem. Soc.</span> <span class="year" tagx="year" title="year">1992</span>, <span class="volume" tagx="volume" title="volume">114</span> <span class="issue" tagx="issue" title="issue">25</span> <span class="fpage" tagx="fpage" title="fpage">10024</span>– <span class="lpage" tagx="lpage" title="lpage">10035</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ja00051a040">10.1021/ja00051a040</a> </span>. </span> </li> <li tag="ref"> <a name="ref62" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riniker</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Landrum</span> <span class="given-names" tagx="given-names" title="given-names">G. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Better Informed Distance Geometry: Using What We Know To Improve Conformation Generation </span>. <span class="source" tagx="source" title="source">J. Chem. Inf. Model. </span> <span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">55</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">2562</span>– <span class="lpage" tagx="lpage" title="lpage">2574</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acs.jcim.5b00654">10.1021/acs.jcim.5b00654</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26575315">26575315</a> </span> </span> </li> <li tag="ref"> <a name="ref63" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alberty</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cornish-Bowden</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goldberg</span> <span class="given-names" tagx="given-names" title="given-names">R. N.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hammes</span> <span class="given-names" tagx="given-names" title="given-names">G. G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tipton</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Westerhoff</span> <span class="given-names" tagx="given-names" title="given-names">H. V.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Recommendations for Terminology and Databases for Biochemical Thermodynamics </span>. <span class="source" tagx="source" title="source">Biophys. Chem.</span> <span class="year" tagx="year" title="year">2011</span>, <span class="volume" tagx="volume" title="volume">155</span> <span class="issue" tagx="issue" title="issue">2–3</span> <span class="fpage" tagx="fpage" title="fpage">89</span>– <span class="lpage" tagx="lpage" title="lpage">103</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bpc.2011.03.007">10.1016/j.bpc.2011.03.007</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21501921">21501921</a> </span> </span> </li> <li tag="ref"> <a name="ref64" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dixon</span> <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jurs</span> <span class="given-names" tagx="given-names" title="given-names">P. C.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Estimation of pKa for Organic Oxyacids Using Calculated Atomic Charges </span>. <span class="source" tagx="source" title="source">J. Comput. Chem.</span> <span class="year" tagx="year" title="year">1993</span>, <span class="volume" tagx="volume" title="volume">14</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">1460</span>– <span class="lpage" tagx="lpage" title="lpage">1467</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/jcc.540141208">10.1002/jcc.540141208</a> </span>. </span> </li> <li tag="ref"> <a name="ref65" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Csizmadia</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tsantili-Kakoulidou</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Panderi</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Darvas</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Prediction of Distribution Coefficient from Structure. 1. Estimation Method </span>. <span class="source" tagx="source" title="source">J. Pharm. Sci.</span> <span class="year" tagx="year" title="year">1997</span>, <span class="volume" tagx="volume" title="volume">86</span> <span class="issue" tagx="issue" title="issue">7</span> <span class="fpage" tagx="fpage" title="fpage">865</span>– <span class="lpage" tagx="lpage" title="lpage">871</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/js960177k">10.1021/js960177k</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9232530">9232530</a> </span> </span> </li> <li tag="ref"> <a name="ref66" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Er</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suh</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marshak</span> <span class="given-names" tagx="given-names" title="given-names">M. P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Computational Design of Molecules for an All-Quinone Redox Flow Battery </span>. <span class="source" tagx="source" title="source">Chem. Sci. </span> <span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">6</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">885</span>– <span class="lpage" tagx="lpage" title="lpage">893</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/C4SC03030C">10.1039/C4SC03030C</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29560173">29560173</a> </span> </span> </li> <li tag="ref"> <a name="ref67" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huskinson</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marshak</span> <span class="given-names" tagx="given-names" title="given-names">M. P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suh</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Er</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerhardt</span> <span class="given-names" tagx="given-names" title="given-names">M. R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galvin</span> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gordon</span> <span class="given-names" tagx="given-names" title="given-names">R. G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aziz</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A Metal-Free Organic-Inorganic Aqueous Flow Battery </span>. <span class="source" tagx="source" title="source">Nature</span> <span class="year" tagx="year" title="year">2014</span>, <span class="volume" tagx="volume" title="volume">505</span> <span class="issue" tagx="issue" title="issue">7482</span> <span class="fpage" tagx="fpage" title="fpage">195</span>– <span class="lpage" tagx="lpage" title="lpage">198</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature12909">10.1038/nature12909</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24402280">24402280</a> </span> </span> </li> <li tag="ref"> <a name="ref68" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tong</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wong</span> <span class="given-names" tagx="given-names" title="given-names">A. A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gómez-Bombarelli</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gordon</span> <span class="given-names" tagx="given-names" title="given-names">R. G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aziz</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">UV-Vis Spectrophotometry of Quinone Flow Battery Electrolyte for in Situ Monitoring and Improved Electrochemical Modeling of Potential and Quinhydrone Formation </span>. <span class="source" tagx="source" title="source">Phys. Chem. Chem. Phys.</span> <span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">19</span> <span class="issue" tagx="issue" title="issue">47</span> <span class="fpage" tagx="fpage" title="fpage">31684</span>– <span class="lpage" tagx="lpage" title="lpage">31691</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/C7CP05881K">10.1039/C7CP05881K</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29165500">29165500</a> </span> </span> </li> <li tag="ref"> <a name="ref69" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tong</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tabor</span> <span class="given-names" tagx="given-names" title="given-names">D. P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beh</span> <span class="given-names" tagx="given-names" title="given-names">E. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Goulet</span> <span class="given-names" tagx="given-names" title="given-names">M.-A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Porcellinis</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gordon</span> <span class="given-names" tagx="given-names" title="given-names">R. G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aziz</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Alkaline Benzoquinone Aqueous Flow Battery for Large-Scale Storage of Electrical Energy </span>. <span class="source" tagx="source" title="source">Adv. Energy Mater.</span> <span class="year" tagx="year" title="year">2018</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">1702056</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/aenm.201702056">10.1002/aenm.201702056</a> </span>. </span> </li> <li tag="ref"> <a name="ref70" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerhardt</span> <span class="given-names" tagx="given-names" title="given-names">M. R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tong</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gómez-Bombarelli</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marshak</span> <span class="given-names" tagx="given-names" title="given-names">M. P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Galvin</span> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gordon</span> <span class="given-names" tagx="given-names" title="given-names">R. G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aziz</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Anthraquinone Derivatives in Aqueous Flow Batteries </span>. <span class="source" tagx="source" title="source">Adv. Energy Mater.</span> <span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">7</span> <span class="issue" tagx="issue" title="issue">8</span> <span class="fpage" tagx="fpage" title="fpage">1601488</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/aenm.201601488">10.1002/aenm.201601488</a> </span>. </span> </li> <li tag="ref"> <a name="ref71" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gómez-Bombarelli</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beh</span> <span class="given-names" tagx="given-names" title="given-names">E. S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tong</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valle</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aspuru-Guzik</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aziz</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gordon</span> <span class="given-names" tagx="given-names" title="given-names">R. G.</span> </span> <span class="mixed-article-title" title="mixed-article-title">A Redox-Flow Battery with an Alloxazine-Based Organic Electrolyte </span>. <span class="source" tagx="source" title="source">Nature Energy</span> <span class="year" tagx="year" title="year">2016</span>, <span class="volume" tagx="volume" title="volume">1</span> <span class="issue" tagx="issue" title="issue">9</span> <span class="fpage" tagx="fpage" title="fpage">16102</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nenergy.2016.102">10.1038/nenergy.2016.102</a> </span>. </span> </li> <li tag="ref"> <a name="ref72" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matthews</span> <span class="given-names" tagx="given-names" title="given-names">A. G. d. G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van der Wilk</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nickson</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fujii</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boukouvalas</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leon-Villagra</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghahramani</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hensman</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">GPflow: A Gaussian Process Library Using TensorFlow </span>. <span class="source" tagx="source" title="source">J. Mach. Learn. Res. </span> <span class="year" tagx="year" title="year">2017</span>, <span class="volume" tagx="volume" title="volume">18</span> <span class="issue" tagx="issue" title="issue">40</span> <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">6</span>. </span> </li> <li tag="ref"> <a name="ref73" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rogers</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hahn</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Extended-Connectivity Fingerprints</span>. <span class="source" tagx="source" title="source">J. Chem. Inf. Model. </span> <span class="year" tagx="year" title="year">2010</span>, <span class="volume" tagx="volume" title="volume">50</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">742</span>– <span class="lpage" tagx="lpage" title="lpage">754</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/ci100050t">10.1021/ci100050t</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20426451">20426451</a> </span> </span> </li> <li tag="ref"> <a name="ref74" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bajusz</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rácz</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Héberger</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Why Is Tanimoto Index an Appropriate Choice for Fingerprint-Based Similarity Calculations? </span>. <span class="source" tagx="source" title="source">J. Cheminf.</span> <span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">20</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s13321-015-0069-3">10.1186/s13321-015-0069-3</a> </span>. </span> </li> <li tag="ref"> <a name="ref75" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elstner</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Porezag</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jungnickel</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elsner</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haugk</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frauenheim</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suhai</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seifert</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Self-Consistent-Charge Density-Functional Tight-Binding Method for Simulations of Complex Materials Properties </span>. <span class="source" tagx="source" title="source">Phys. Rev. B: Condens. Matter Mater. Phys. </span> <span class="year" tagx="year" title="year">1998</span>, <span class="volume" tagx="volume" title="volume">58</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">7260</span>– <span class="lpage" tagx="lpage" title="lpage">7268</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1103/PhysRevB.58.7260">10.1103/PhysRevB.58.7260</a> </span>. </span> </li> <li tag="ref"> <a name="ref76" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sure</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grimme</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Corrected Small Basis Set Hartree-Fock Method for Large Systems </span>. <span class="source" tagx="source" title="source">J. Comput. Chem.</span> <span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">34</span> <span class="issue" tagx="issue" title="issue">19</span> <span class="fpage" tagx="fpage" title="fpage">1672</span>– <span class="lpage" tagx="lpage" title="lpage">1685</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/jcc.23317">10.1002/jcc.23317</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23670872">23670872</a> </span> </span> </li> <li tag="ref"> <a name="ref77" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grimme</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brandenburg</span> <span class="given-names" tagx="given-names" title="given-names">J. G.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bannwarth</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hansen</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Consistent Structures and Interactions by Density Functional Theory with Small Atomic Orbital Basis Sets </span>. <span class="source" tagx="source" title="source">J. Chem. Phys.</span> <span class="year" tagx="year" title="year">2015</span>, <span class="volume" tagx="volume" title="volume">143</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">054107</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1063/1.4927476">10.1063/1.4927476</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26254642">26254642</a> </span> </span> </li> <li tag="ref"> <a name="ref78" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kozuch</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">J. M. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">DSD-PBEP86: In Search of the Best Double-Hybrid DFT with Spin-Component Scaled MP2 and Dispersion Corrections </span>. <span class="source" tagx="source" title="source">Phys. Chem. Chem. Phys.</span> <span class="year" tagx="year" title="year">2011</span>, <span class="volume" tagx="volume" title="volume">13</span> <span class="issue" tagx="issue" title="issue">45</span> <span class="fpage" tagx="fpage" title="fpage">20104</span>– <span class="lpage" tagx="lpage" title="lpage">20107</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/c1cp22592h">10.1039/c1cp22592h</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21993810">21993810</a> </span> </span> </li> <li tag="ref"> <a name="ref79" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riplinger</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neese</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="mixed-article-title" title="mixed-article-title">An Efficient and near Linear Scaling Pair Natural Orbital Based Local Coupled Cluster Method </span>. <span class="source" tagx="source" title="source">J. Chem. Phys.</span> <span class="year" tagx="year" title="year">2013</span>, <span class="volume" tagx="volume" title="volume">138</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">034106</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1063/1.4773581">10.1063/1.4773581</a> </span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23343267">23343267</a> </span> </span> </li> <li tag="ref"> <a name="ref80" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boudart</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Thermochemical Kinetics, 2nd. Ed., Sidney W. Benson, Wiley Interscience, 320, New York, 1976 </span>. <span class="source" tagx="source" title="source">AIChE J.</span> <span class="year" tagx="year" title="year">1977</span>, <span class="volume" tagx="volume" title="volume">23</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">613</span>– <span class="lpage" tagx="lpage" title="lpage">613</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/aic.690230437">10.1002/aic.690230437</a> </span>. </span> </li> </ul> </div> </div>  </body></html>